codespell_lib/data/dictionary.txt

```diff
@@ -12201,7 +12201,7 @@ overwite->overwrite
 overwites->overwrites
 overwitten->overwritten
 overwize->otherwise
-overwridden->overwritten
+overwridden->overwritten, overridden,
```

Comment:
I think overridden should be the first suggestion!

Reply:
2982c26